### PR TITLE
Cover demo programs in primary README

### PR DESCRIPTION
**Pull request by dwalton76**
Other small changes:
- demo/EV3RSTORM/ev3rstorm.py was not executable
- demo/MINDCUB3R/__init__.py was not needed

For issue #273

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -126,6 +126,9 @@ for ev3dev. The first allows you to run this program from Brickman, while the
 second imports this library.
 
 When saving Python files, it is best to use the ``.py`` extension, e.g. ``my-file.py``.
+Your program must be executable. To mark a program as executable run ``chmod +x my-file.py``
```
Comment (WasabiFan) [inline README.rst]: Can you add a lead-in sentence/phrase here to make it clear that this is all part of the same idea? e.g. "To be able to run your Python code, your program must be executable."

File: demo/README.md [hunk 2]
```diff
@@ -1,13 +1,35 @@
 # ev3dev demo programs
 
-This folder contains a bunch of demo programs that you can use to help you in
+This folder contains several demo programs that you can use to help you in
 developing your own code. Brief descriptions of each demo are provided below;
 you can access the full source code and some more detailed information on each
 by opening the respective folders above.
 
+To install these on your EV3, use git to clone the ev3dev-lang-python repository.
```
Comment (WasabiFan) [inline demo/README.md]: Can you add a note that this requires internet access?

Comment (WasabiFan) [inline demo/README.md]: Given that we're presenting two independent options here, can you start this with something like "There are two options for running these demo programs. We recommend that you..."?

Comment (WasabiFan) [inline demo/README.md]: You'd then add an equivalent sentence introducing the second option. Maybe even with headings.

**Comment by dwalton76** [comment]
Sorry, I forgot all about this pull requests :( Will update it to address @WasabiFan 's feedback

**Comment by dwalton76** [comment]
I hate git sometimes...no idea why all of those other commits are showing up in the pull request.  Let me close this one, create a new branch and submit a new pull request to clean this up.

**Pull request closed without merging**